cms-xcache/limits.d/50-xrootd.conf

```diff
@@ -0,0 +1,5 @@
+xrootd     soft    nproc     20000
+xrootd     hard    nproc     21000
+xrootd     soft    nofile    99000
+xrootd     hard    nofile    100000
+*          soft    core      unlimited
```

Comment:
Can we rename this file to `50-cms-xcache-limits.conf`?

Reply:
No objections.

Reply:
Renamed.